File: 프로그래머스/2/64065. 튜플/튜플.js
function solution(s) {
    let dict = {}
    for (let v of s.match(/\d+/g)) dict[v] = (dict[v] || 0) + 1;
    
    return Object.keys(dict).map(v => +v).sort((a, b) => dict[b] - dict[a]);
}

File: 프로그래머스/2/64065. 튜플/튜플.py
def solution(s):
    answer = []
    s = s[2:-2].split("},{")
    s = sorted(s, key = len)
    
    for v in s:
        numbers = v.split(",")
        for n in numbers:
            if int(n) not in answer: answer.append(int(n))
            
    return answer